setup.cfg

```diff
@@ -44,6 +44,7 @@ install_requires=
     keyring >= 15.1
 setup_requires =
     setuptools_scm >= 1.15
+zip_safe = False
```

Comment:
The mypy docs (and some closed issues) suggest adding this, but cursory testing suggests it might not be necessary. I opted to be conservative and leave it it.

Reply:
Please remove this. It's cruft that I'm trying to discourage users from using. Most packages should aim to be zip safe anyway, and if installed using pip, it won't make a difference. Plus, mypy should support zip-safe packages.

Reply:
To get some more clarity, I opened https://github.com/python/mypy/issues/8802. I'm going to wait a day or two to see if anyone chimes in, otherwise I'll happily remove it.

Reply:
@jaraco Any thoughts based on the comments in the mypy issue? Based on my limited understanding, I agree that this seems mostly unnecessary, but also maybe mostly harmless?

I'm game to update the mypy docs as suggested (and maybe prevent the further spread of `zip_safe = False`), but I'd like to get a more concrete understanding first.

Reply:
The harm comes from adding unnecessary cruft and creating a model that other users may copy. I'd like this project to be a model of excellence that other projects should feel comfortable copying. I added some comments there, but I'm 99% certain there's no unmitigatable harm in twine omitting this setting.

Reply:
BTW, thanks for running this to ground rather than just addressing it here. I know that's a lot of effort... to find the right solution, and I appreciate it very much.